src/event_ffi.mjs

```diff
@@ -1,3 +1,22 @@
 export function target(event) {
   return event.target;
 }
+
+export function constructorName(value) {
+  return value.constructor.name;
+}
+
+export function toKeyboardEventUnsafe(event) {
+  return {
+    event,
```

Comment:
I don't think the full event should be returned. users can just keep a reference to what they passed in.